src/test/test_dataset_utils.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
+
+from dataset.dataset_utils import VideosIndexer
+
+
+class TestVideosIndexer(TestCase):
+    def setUp(self):
+        self.video_list = [["a", "b", "c"], ["d"], ["e", "f"]]
+        self.video_indexer = VideosIndexer(self.video_list)
+
+    def test_get_video_OutOfBounds(self):
+        self.assertRaises(ValueError, self.video_indexer.get_video_frame, -1)
```

Comment:
The motivation for the ValueError note that I wrote above has to do with this. This test does not verify which of the two `ValueError` were raised. If you add a message, you can use [assertRaisesRegex](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex) to verify that the behavior is the expected one.